python/ad-hoc-ingest/README.md

```diff
@@ -0,0 +1,66 @@
+# Ad hoc Ingest
+
+This directory contains the scripts used for ingesting the data from ad hoc sources such as hard drives that 
+The National Archives has received over the years. The scripts and process strictly applies to the data 
+which meets the following criteria:
+- The data has been already received and held on some storage media 
+- The data is NOT ingested in the existing DRI system
+- The data has been already available to the public on Discovery.
+
+## The process:
+![adhoc-ingest-flow.png](images/adhoc-ingest-flow.png)
+
+There is a script at the centre of the ingest process which drives the generic ingest. The script receives a CSV file as an input. 
+This CSV file contains the filenames, catalog reference, any other information and a checksum. The files are 
+located in a folder or other media which can be made accessible to the script. The script works chronologically n steps 
```

Comment:
1. Maybe "diagram" rather than "picture"?
2. Is this supposed to say "n steps" (like "n" for "number") or "in steps"? If the former, maybe "The order in which this script is run is detailed on the diagram"